packages/askui-nodejs/src/core/ui-control-commands/command-data.ts

```diff
@@ -0,0 +1,15 @@
+import { ControlCommand } from './control-command';
+import { ControlCommandCode } from './control-command-code';
+
+export class CommandData {
+  constructor(
+    public type:string,
+    public data: ControlCommand [] = [],
+  ) {}
+
+  static getControlCommandFromJson(json:CommandData, resizeRatio = 1) {
+    return ControlCommand
+      .fromJson(json.data[0]
```

Comment:
what is this 0?

Reply:
I put the ControlCommand in an data array in the backend or not 100% sure what is your question?

Reply:
the controlcommand can be be an array and not one single command